databroker/core.py

```diff
@@ -161,8 +162,8 @@ def _create_open_args(self, user_parameters):
         open_args['entry'] = self
         return plugin, open_args
 
-    def cache_clear(self):
-        self.__cache.clear()
+    def _get_cached(self, **kwargs):
```

Comment:
I think this change might be accidental. We don't need `_get_cached` (that was from an early draft of #494) and I think we _do_ want `cache_clear`.

Reply:
fixed that one already